### DISCUSSION TITLE
Just ask: is this based on redbean?

### DISCUSSION THREAD

**Discussion by happy15** [sqlpage/SQLPage#277] [Q&A] [answered]
I see the lobster icon when launching sqlpage.bin, so I thought sqlpage is based on redbean.

BTW, cool project and I can instantly launch a site based on my existing sqlite db.

👍

**Comment by lovasoa** [comment, marked as answer]
Hello, no, this is not based on redbean (even though I'm a fan of everything Justine Tunney makes). 

Can you post a screenshot of the lobster you are seeing? SQLPage is not supposed to display a lobster anywhere...

**Reply by happy15** [reply]
I asked a dumb question, as you already provided different binaries for different os platform.

It must be some browser cache problem. Never mind and thanks for the reply.
